bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -699,6 +727,88 @@ private int updateFromSchema3toSchema4(int schemaVersionNumber, CaseDbConnection
 	}
 
 	/**
+	 * Updates a schema version 3 database to a schema version 4 database.
+	 *
```

Comment:
Please fix the copy-paste error here (version 4 to 5 upgrade, not version 3 to 4).